src/app/qgisapp.cpp

```diff
@@ -4464,6 +4464,8 @@ void QgisApp::addGeonodeLayer()
   }
   connect( geonodes, SIGNAL( addRasterLayer( QString const &, QString const &, QString const & ) ),
            this, SLOT( addRasterLayer( QString const &, QString const &, QString const & ) ) );
+  connect( geonodes, SIGNAL( addWfsLayer( QString, QString, QString ) ),
```

Comment:
Ah shame, I didn't check that one. I was focusing to fix the new slot. Ok, I will fix it. Thanks @Gustry